testsuite/features/secondary/buildhost_docker_build_image.feature

```diff
@@ -81,6 +98,8 @@ Feature: Build container images
     And I select the hostname of "build_host" from "buildHostId"
     And I click on "submit-btn"
     Then I wait until I see "GUI_BUILT_IMAGE" text
```

Comment:
this line is shown also when the image is still compiling. We need to wait "GUI_BUILT_IMAGE" should contain "1"